podcasts/Profile - SwiftUI/Account/AccountHeaderView.swift

```diff
@@ -0,0 +1,148 @@
+import SwiftUI
+import PocketCastsUtils
+import PocketCastsServer
+
+struct AccountHeaderView: View {
+    @EnvironmentObject var theme: Theme
+    @ObservedObject var viewModel: AccountHeaderViewModel
+
+    var body: some View {
+        container { proxy in
+            SubscriptionProfileImage(viewModel: viewModel)
+                .frame(width: Constants.imageSize, height: Constants.imageSize)
+
+            ProfileInfoLabels(profile: viewModel.profile, alignment: .center, spacing: Constants.spacing)
+                .padding(.top, 5)
```

Comment:
Nit picky: Some size constants are being stored in the `Constants` enum and some aren't.  Might be nice if that was consistent

Reply:
Good catch, I've updated it here: 711f703f1048c3de332f815a7ef6f157dca3451f